### PR TITLE
Feat/promises

### PR DESCRIPTION
**Pull request by anneback**

### CODE REVIEW

File: src/utils/api-fake.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+// `by`—The username of the item's author.
```
Comment (iamstarkov) [inline src/utils/api-fake.js]: you don't use this file

File: src/utils/api.js [hunk 2]
```diff
@@ -1,135 +1,24 @@
...
+  return fetchJson(`${BASE}/v0/topstories.json`).then(res => {
+    return first10(res);
+  });
+  // TODO: fetch the endpoint, get first 10, return it
 };
+
+// Item endpoint https://hacker-news.firebaseio.com/v0/item/8863.json
+const getItem = id => {
+  return fetchJson(`${BASE}/v0/item/${id}.json`);
+  // TODO: fetch relevant item, return it
```
Comment (iamstarkov) [inline src/utils/api.js]: these TODOs are done, shall it be removed?

**Review by iamstarkov — APPROVED** [review]
there is dead code and comments, which would be nice to remove before merge, but LGTM

File: src/utils/api.js (hunk 2 above)
Comment (anneback) [reply src/utils/api.js]: I saved this during implementation in order to not have to swap between windows :P.

File: src/utils/api-fake.js (hunk 1 above)
Comment (anneback) [reply src/utils/api-fake.js]: I'll remove this.

**Pull request merged**